velox/functions/sparksql/fuzzer/SparkAggregationFuzzerTest.cpp

```diff
@@ -82,6 +82,12 @@ int main(int argc, char** argv) {
           {"skewness", nullptr},
           {"kurtosis", nullptr},
           {"collect_list", makeArrayVerifier()},
+          // The Ascending sorting order used  'min' by Spark places nulls
+          // first.
+          {"min", nullptr},
+          // The Ascending sorting order used  'max' by Spark places nulls
+          // first.
+          {"max", nullptr},
```

Comment:
We need it because the result is different with DuckDB's, right?

Reply:
Yes, this is because null order is different.